### ISSUE TITLE
Datsemshifts Database: new concept list extracted from the 2020 version

### ISSUE THREAD

**Issue by LinguList** [concepticon/concepticon-data#954]
I just managed to extract the most recent version of the [database of semantic shifts](http://datsemshift.ru/). 

I'd quote this as:
```
@Book{Zalizniak2020,
  address   = {Moscow},
  author    = {Zalizniak, Anna A. and Smirnitskaya, Anna and Russo, Maksim and Mikhailova, Tatiana and Bobrik, Marina and Gruntov, Ilya and Orlova, Maria and Bibaeva, Maria and Voronov, Mikhail},
  publisher = {Institute of Linguistics at the Russian Academy of Sciences},
  title     = {Database of Semantic Shifts (Version from 07/10/2020)},
  year      = {2020},
  url       = {http://datsemshift.ru/},
  version   = {2020-10-07},
  timestamp = {2020.10.07},
}
```

I have assembled all data (and can also do so in new uploads, provided the website does not change), and suggest that these be mapped. 

[Zalizniak-2020-2578.tsv.zip](https://github.com/concepticon/concepticon-data/files/5339798/Zalizniak-2020-2578.tsv.zip)

The URLs are compose of "http://datsemshift.ru/{SHIFT}", this should be added to the metadata.json. I also think it is impossible to link all 2500+x concepts, but it looks like it is better to try and do so manually than doing this automatically only in NoRaRe. 

@AnnikaTjuka, do you want to have a look at this later, or should we ask somebody else?

**Comment by AnnikaTjuka** [comment]
Thanks! Yes, I can do the mappings. It will take a bit of time but I already saw many body parts and can look at their semantic shifts more closely :)

**Comment by LinguList** [comment]
Nice. I think this list is in fact quite valuable, also for comparison
with CLICS. It can come in handy in future studies, also in our norare
revision, since it is yet another cross-linguistic dataset.

**Comment by AnnikaTjuka** [comment]
I had a first look at the data. Some meanings occur multiple times, e.g., _already_ (sometimes with different writing styles: `<animal>` and `animal`). @LinguList Should I avoid double-mappings or is it OK to map the same gloss multiple times?

**Comment by LinguList** [comment]
I'd systematically ignore all `<>` cases, as they look like some weird internal annotation to me. This could then also be mentioned in the NOTE.

**Comment by AnnikaTjuka** [comment]
Ok, I'll ignore the `<>` cases. But can I map multiple occurrences of the same gloss? For example, `anger` occurs twice without differences in writing.

**Comment by LinguList** [comment]
How many cases do we have of this kind? I cannot confirm this for "anger", for example. I only see one occurrence: 1009

**Comment by LinguList** [comment]
Ah, and what is problematic as well is that I added one empty line (this is probably  better to be removed). Maybe, we can also make a deliberate cut-off of data poitns? If there are two angers, for example (although I didn't find them), do both have data? Then one could pick one with data. 

But if there are many synonyms, I should maybe try to infer them automatically beforehand?

**Comment by AnnikaTjuka** [comment]
> How many cases do we have of this kind? I cannot confirm this for "anger", for example. I only see one occurrence: 1009

I saw _anger_ twice on the [website](http://datsemshift.ru/meanings/a) and assumed that it would occur in the data set as well. But I just checked the list and only the ones with `<>` occur twice. So we do not need to worry about double-mapping if we exclude the `<>` cases.

Regarding synonyms, there might be a few (e.g., _24 hours_ and _day_) but at first glance, most glosses seem to be straightforward. I'll start mapping now to get a better grasp of the list content.
